### DISCUSSION TITLE
Can't activate the SHARPy conda environment

### DISCUSSION THREAD

**Discussion by caiomaxdias** [ImperialCollegeLondon/sharpy#215] [Q&A] [answered]
**Description of the problem**
I've been trying to install the SHARPy by a Docker Container following all the steps of the installation guide, but when I tried to activate the SHARPy conda environment (by using: 'conda activate sharpy_env') this message appeared on the screen:
_Could not find conda enviroment: sharpy_env
You can list all discoverable environments with 'conda info --envs'_ 

In another hand, i looked for another alternative by typing 'conda activate sharpy_minimal' and anything appeared (I thought it finally worked, but i'm not sure of that). So I continued, but when I try to run a Sharpy case, I'm not able to create a generate_case file following the provided templates: 

(sharpy_env) python generate_case.py  

When i used this template, i received this message:  _bash: syntax error near unexpected token 'python'_

**System Info:**
 - OS: [e.g. Windows 10]
 - SHARPy Version [e.g. v1.2]

**Additional context**
I need assistance, hope somebody answer me.

**Comment by ngoiz** [comment, marked as answer]
Hello,

Running the container image already activates the `sharpy_minimal` environment so the code is ready to go. I just checked and everything runs fine. EDIT: The Docker build of SHARPy has been updated and available [here](https://github.com/ImperialCollegeLondon/sharpy/pkgs/container/sharpy) (See #218 for details).

The environment also includes python so just running `python` from within the container will get you a python prompt. I am unsure about your error message. Make sure you are running a known case file from the appropriate directory.

I am changing this issue to a discussion as it is not a bug. We will update the SHARPy version on docker soon though.
